iotdb-core/mlnode/iotdb/mlnode/log.py

```diff
@@ -17,18 +17,106 @@
 #
 import logging
 import os
-from logging.config import fileConfig
+import random
+import sys
+import multiprocessing
+import inspect
 
-from iotdb.mlnode.constant import (MLNODE_CONF_DIRECTORY_NAME,
-                                   MLNODE_LOG_CONF_FILE_NAME)
+# log directory
+LOG_DIR = "./logs"
+# level fot stdout
+STD_LEVEL = logging.INFO
 
-log_conf_file = os.path.join(os.getcwd(), MLNODE_CONF_DIRECTORY_NAME, MLNODE_LOG_CONF_FILE_NAME)
-if os.path.exists(log_conf_file):
-    fileConfig(log_conf_file)
-else:
-    logging.basicConfig(
-        level=logging.DEBUG,
-        format='%(asctime)s - %(name)s - %(levelname)s - %(message)s'
-    )
 
-logger = logging.getLogger()
+class LoggerFilter(logging.Filter):
+    def filter(self, record):
+        record.msg = f"{self.custom_log_info()} Message: {record.msg}"
+        return True
+
+    @staticmethod
+    def custom_log_info():
+        frame = inspect.currentframe()
+        stack_trace = inspect.getouterframes(frame)
+
+        pid = os.getpid()
+        process_name = multiprocessing.current_process().name
+
+        # 格式化函数调用栈信息
+        stack_info = ""
+        frame_info = stack_trace[7]  # 忽略前两个栈帧
```

Comment:
Please change the comments to English.

Reply:
We are already aware of these issues, and these fixes are already in our plan. We will correct these obvious deficiencies in subsequent commits